tools/utility/yarn.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import json
+import os
+
+from termcolor import colored
+
+
+def find_eslint(node_modules_path):
+    # Check in the node_modules folder for eslint
+    eslint_path = os.path.join(node_modules_path, ".bin", "eslint")
+    if os.path.exists(eslint_path):
+        return eslint_path
+
+    print(
+        colored(
+            "Cannot find eslint, run `./b yarn` first if you need to format typescript files", "red", attrs=["bold"]
+        )
+    )
+    return None
+
+
+def find_package_json(project_root=".", package_name="NUsight"):
```

Comment:
This function is probably doing unnecessary work. We can start by just checking that `nusight2/package.json` exists. It's unlikely that it won't be there.

Reply:
That is literally the first thing this function does

Reply:
You mean after this commit? [`cafae04` (#361)](https://github.com/NUbots/NUbots/pull/361/commits/cafae049a6dc94b80722c4605a218855d843076c) Or before?

Reply:
Oh, nevermind then :P